src/lfsr/lfsr_galois.veryl

```diff
@@ -0,0 +1,168 @@
+module lfsr_galois #(
```

Comment:
```suggestion
pub module lfsr_galois #(
```

The exported modules should have `pub`.
https://doc.veryl-lang.org/book/04_language_reference/11_visibility.html